easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-3.6.11-goolf-1.4.10.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'MIGRATE-N'
+version = '3.6.11' 
+
+homepage = 'http://popgen.sc.fsu.edu/Migrate/Migrate-n.html'
+description = """ Migrate estimates population parameters, effective population sizes
+ and migration rates of n populations, using genetic data.  It uses a coalescent theory 
+ approach taking into account history of mutations and uncertainty of the genealogy. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://popgen.sc.fsu.edu/currentversions/']
+sources = ['migrate-%(version)s.src.tar.gz']
+
+start_dir = "src"
+
+parallel = 1
+
+with_configure = True
+
```

Comment:
`configopts = '--enable-A4 '` because the default page size is letter (although this is my european preference....)